src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -712,6 +715,49 @@ bool CountryInstance::apply_history_to_country(
 	for (auto const& [technology, level] : entry.get_technologies()) {
 		ret &= set_technology_unlock_level(*technology, level);
 	}
+
+	const auto inform_limit_parsing_failed = [](Invention const& invention) {
+		Logger::warning("Failed to parse limit for invention ", invention.get_identifier());
+	};
+
+	for (Invention const& invention : invention_manager.get_inventions()) {
+		ConditionNode const& root_condition_node = invention.get_limit().get_condition_root();
+		Condition const* const root_condition_ptr = root_condition_node.get_condition();
+		if(root_condition_ptr == nullptr) {
+			continue;
+		}
+		Condition const& root_condition = *root_condition_ptr;
+		if(root_condition.get_value_type() != value_type_t::GROUP
+			|| root_condition.get_identifier() != "AND") {
```

Comment:
```suggestion
		if (
			root_condition.get_value_type() != value_type_t::GROUP
			|| root_condition.get_identifier() != "AND"
		) {
```
As a general rule of the formatting standard in this repo: if there's a line break anywhere between a pair of brackets (whether `()`, `{}` or `[]`) then we also break after the opening bracket and before the closing bracket. So either everything's on a single line or we have something like:
```c++
blah(
	...
)
```